common/daq/generation/gen_embedded_can.py

```diff
@@ -201,14 +245,19 @@ def configure_node(node_config, node_paths):
 
     # Send Macros, requires knowledge of CAN peripheral
     macro_lines = []
-    periph = DEFAULT_PERIPHERAL
+    periph = DEFAULT_PERIPHERAL_NODE
     if 'can_peripheral' in node_config: periph = node_config['can_peripheral']
     for msg in node_config['tx']:
+        periph = DEFAULT_PERIPHERAL_NODE
+
+        if 'can_peripheral_override' in msg:
+                periph = msg['can_peripheral_override']
```

Comment:
Indentation seems a bit too deep. I have no idea if python will get mad at this or not.